app/addons/setup/__tests__/optionalSettings-test.js

```diff
@@ -0,0 +1,48 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
Could you rename this to `optionalSettings.test`. Ideally all our jest tests have the `.test.js` extension.